app/src/main/java/com/elmeradrianv/shesafe/MainActivity.java

```diff
@@ -45,8 +46,8 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                                 //fragment = fragment2;
```

Comment:
let's delete this too. You can always add a toast here that says "Not yet implemented" if you want something there.